camkes-vm/build.py

```diff
@@ -33,6 +33,10 @@ def run_build(manifest_dir: str, build: Build):
     if plat.arch == 'x86':
         del build.settings['PLATFORM']  # not used for x86 in this test, avoid warning
 
+    # if vm_platform is set, the init-build.sh script expects a different platform name.
+    if build.vm_platform:
```

Comment:
Does this require any update the `Build` class in `seL4-platforms/builds.py` then also? Seems any attributes from the YAML file are put into the build object, but there is no check if this potentially clashes somewhere?